### PR TITLE
Adding Alpine 3.8 and 3.9 to the RID graph

### PR DESCRIPTION
**Pull request by joperezr**
Fixes: #34316

#### Description
Our Docker images have now moved to Alpine 3.8 and with .NET Core 2.1 and 2.2 we claim to support 3.7+. However, our RID graphs only have Alpine 3.7 on the graph, which means that whenever a customer has some special native assets for 3.8 they won't be selected correctly and customers may run into unexpected runtime issues. This change will add both Alpine 3.8 and 3.9 to the RID graph.
#### Customer Impact
If a customer uses one of our docker images, or is running Alpine 3.8, and want to use special native assets for the new version of the distro they will hit issues as their assets won't be correctly selected. There is a way to work around this for now by setting an environment variable (`DOTNET_RUNTIME_ID=linux-musl-x64`) but the proper fix is to add them to the RID graph.
#### Regression?
No
#### Packaging reviewed? 
No
#### Risk
Low. We have added RIDs many times in the past and it is rarely an issue.

cc: @eerhardt @MichaelSimons

### CODE REVIEW

File: pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json [hunk 1]
```diff
@@ -1,8492 +1,8659 @@
 {
-  "alpine": [
```
Comment (eerhardt) [inline pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json]: what happened with the whitespace in this file?

Comment (joperezr) [reply pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json]: oh, not sure, I opened it and edited it in VS, so perhaps it corrected the spacing. I can revert that if we want to.

Comment (eerhardt) [reply pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json]: These 2 json file should be generated using the `/p:UpdateRuntimeFiles=true` property. Are you using that?

https://github.com/dotnet/corefx/tree/master/pkg/Microsoft.NETCore.Platforms#adding-a-new-os

Comment (joperezr) [reply pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json]: Ok, I didn't know you could do that, I just did and updated the PR.

**Comment by joperezr** [comment]
test failures are not related to my PR.

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by jamshedd** [comment]
Approved for 2.2.3.

**Comment by eerhardt** [comment]
> can we define a RID that covers all Alpine 3.x so we don't have to do this whenever they increment their minor versoin?

See https://github.com/dotnet/corefx/issues/34316#issuecomment-451485481 for 2 approaches that will fix this better in the future.

**Comment by danmosemsft** [comment]
Branch not quite open yet

**Comment by karelz** [comment]
@jamshedd this is PR in 2.1 branch, not 2.2 -- fixing milestone to 2.1.9 (I assume that's where this fix got).

**Pull request merged**